diff -up Candle-1.2b/src/frmabout.cpp.1~ Candle-1.2b/src/frmabout.cpp
--- Candle-1.2b/src/frmabout.cpp.1~	2019-10-30 16:40:25.000000000 +0100
+++ Candle-1.2b/src/frmabout.cpp	2020-09-11 15:35:23.587906180 +0200
@@ -12,8 +12,11 @@ frmAbout::frmAbout(QWidget *parent) :
     ui->setupUi(this);
 
     ui->lblAbout->setText(ui->lblAbout->text().arg(qApp->applicationVersion()));
-
+#ifdef UNIX
+    QFile file("/usr/share/doc/" + qApp->applicationDisplayName() + "/LICENSE");
+#else
     QFile file(qApp->applicationDirPath() + "/LICENSE");
+#endif
 
     if (file.open(QIODevice::ReadOnly)) {
         ui->txtLicense->setPlainText(file.readAll());
diff -up Candle-1.2b/src/frmmain.cpp.1~ Candle-1.2b/src/frmmain.cpp
--- Candle-1.2b/src/frmmain.cpp.1~	2020-09-11 15:35:23.588906195 +0200
+++ Candle-1.2b/src/frmmain.cpp	2020-09-11 17:36:15.772805181 +0200
@@ -32,6 +32,8 @@
 #include <QAction>
 #include <QLayout>
 #include <QMimeData>
+#include <QStandardPaths>
+
 #include "frmmain.h"
 #include "ui_frmmain.h"
 #include "ui_frmsettings.h"
@@ -116,7 +118,11 @@ frmMain::frmMain(QWidget *parent) :
     m_spindleCommandSpeed = false;
 
     // Loading settings
+#ifdef UNIX
+    m_settingsFileName = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + "/candle/settings.ini";
+#else
     m_settingsFileName = qApp->applicationDirPath() + "/settings.ini";
+#endif
     preloadSettings();
 
     m_settings = new frmSettings(this);
diff -up Candle-1.2b/src/main.cpp.1~ Candle-1.2b/src/main.cpp
--- Candle-1.2b/src/main.cpp.1~	2019-10-30 16:40:25.000000000 +0100
+++ Candle-1.2b/src/main.cpp	2020-09-11 15:35:23.588906195 +0200
@@ -44,7 +44,11 @@ int main(int argc, char *argv[])
 //    QLocale::setDefault(QLocale("es"));
 
     QString loc = QLocale().name().left(2);
+#ifdef UNIX
+    QString translationsFolder = "/usr/share/" + qApp->applicationDisplayName() + "/translations/";
+#else
     QString translationsFolder = qApp->applicationDirPath() + "/translations/";
+#endif
     QString translationFileName = translationsFolder + qApp->applicationDisplayName() + "_" + loc + ".qm";
 
     qDebug() << "locale:" << loc;
